test/extended/default_build_labels_test.go

```diff
@@ -0,0 +1,139 @@
+// +build default
```

Comment:
the file name should be test/extended/builds/labels.go (after the other PR is merged)